doc/env.md

```diff
@@ -0,0 +1,184 @@
+| Environment Variable | `FLUSH_INTERVAL` |
+| Type | `number` |
+| Required | no |
+| Default | `1000` |
+
+***
+
+### `FLUSH_LIMIT`
+
+> If the length of the send buffer exceeds this length, send immediately
+
+| Config | Value |
+| --- | --- |
+| Name | `flush-limit` |
+| Environment Variable | `FLUSH_LIMIT` |
+| Type | `number` |
+| Required | no |
+| Default | `25` |
+
+***
+
+### `HOSTNAME`
+
+> Optionally, use alternative host name set through the environment
+
+| Config | Value |
+| --- | --- |
+| Name | `hostname` |
+| Environment Variable | `HOSTNAME` |
+| Type | `string` |
+| Required | no |
+| Default | `(none)` |
+
+***
+
+### `HTTP_PROXY`
+
+> An http:// proxy URL to pass through before going to LogDNA
+
+| Config | Value |
+| --- | --- |
+| Name | `http-proxy` |
+| Environment Variable | `HTTP_PROXY` |
+| Type | `string` |
+| Required | no |
+| Default | `(none)` |
+
+***
+
+### `HTTPS_PROXY`
+
+> A secure (https://) proxy URL to pass through before going to LogDNA
+
+| Config | Value |
+| --- | --- |
+| Name | `https-proxy` |
+| Environment Variable | `HTTPS_PROXY` |
+| Type | `string` |
+| Required | no |
+| Default | `(none)` |
+
+***
+
+### `INGESTION_ENDPOINT`
+
+> The endpoint for log ingestion at LogDNA
+
+| Config | Value |
+| --- | --- |
+| Name | `ingestion-endpoint` |
+| Environment Variable | `INGESTION_ENDPOINT` |
+| Type | `string` |
+| Required | no |
+| Default | `/logs/ingest` |
+
+***
+
+### `INGESTION_HOST`
+
+> The host for log ingestion
+
+| Config | Value |
+| --- | --- |
+| Name | `ingestion-host` |
+| Environment Variable | `INGESTION_HOST` |
+| Type | `string` |
+| Required | no |
+| Default | `logs.logdna.com` |
+
+***
+
+### `INGESTION_KEY`
+
+> LogDNA Ingestion Key to stream the logs from files
+
+| Config | Value |
+| --- | --- |
+| Name | `ingestion-key` |
+| Environment Variable | `INGESTION_KEY` |
+| Type | `string` |
+| Required | **yes** |
+| Default | `(none)` |
+
+***
+
+### `INGESTION_PORT`
+
+> The port for log ingestion
+
+| Config | Value |
+| --- | --- |
+| Name | `ingestion-port` |
+| Environment Variable | `INGESTION_PORT` |
+| Type | `number` |
+| Required | no |
+| Default | `443` |
+
+***
+
+### `PROXY`
+
+> A full proxy URL (including protocol) to pass through before going to LogDNA
+
+| Config | Value |
+| --- | --- |
+| Name | `proxy` |
+| Environment Variable | `PROXY` |
+| Type | `string` |
+| Required | no |
+| Default | `(none)` |
+
+***
+
+### `SSL`
+
+> Use https:// for log ingestion
+
+| Config | Value |
+| --- | --- |
+| Name | `ssl` |
+| Environment Variable | `SSL` |
+| Type | `boolean` |
+| Required | no |
+| Default | `true` |
+
+***
+
+### `TAGS`
+
+> Optionally, use comma-separated tags set through the environment
+
+| Config | Value |
+| --- | --- |
+| Name | `tags` |
+| Environment Variable | `TAGS` |
+| Type | `string` |
+| Required | no |
+| Default | `(none)` |
+
+***
+
+### `USER_AGENT`
+
+> user-agent header value to use while sending logs
+
```

Comment:
To avoid having a lower-case at beginning of this phrase, let's say "The user-agent header value..."

Reply:
just a note, this file shouldn't be edited directly - this can be changed in `config.js` and then regenerate

Reply:
Yup, @mdeltito, that's how I have just updated!